server_api_swagger.json

```diff
@@ -181,13 +181,6 @@
             "name": "file",
             "in": "formData",
             "required": true
-          },
-          {
-            "type": "string",
-            "description": "associated typeID for the layout",
-            "name": "classifierId",
```

Comment:
why was this removed?

Reply:
removed in https://github.com/demisto/server/commit/9bbf53f54c54abab4a8895f01de69e5de706c51d
@aviadl can you suggest here please?